sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
-## 1.1.0-beta.4 (Unreleased)
+## 1.1.0 (2021-11-16)
```

Comment:
I think we need release a beta version first where people can try for at least 1 month, and the pump up the version to GA.
https://azure.github.io/azure-sdk/policies_releases.html

![image](https://user-images.githubusercontent.com/2148028/141921412-20e8a0b7-f896-44bb-8b0d-0cfae586e7db.png)

Reply:
Sure. I'll update.